charts/airbyte/templates/server/deployment.yaml

```diff
@@ -39,18 +39,18 @@ spec:
         - name: AIRBYTE_VERSION
           valueFrom:
             configMapKeyRef:
-              name: airbyte-env
+              name: {{ include "common.names.fullname" . }}-env
               key: AIRBYTE_VERSION
         - name: CONFIG_ROOT
           valueFrom:
             configMapKeyRef:
-              name: airbyte-env
+              name: {{ include "common.names.fullname" . }}-env
               key: CONFIG_ROOT
         {{- if .Values.postgresql.enabled }}
         - name: DATABASE_PASSWORD
           valueFrom:
-            configMapKeyRef:
-              name: airbyte-env
+            secretKeyRef:
+              name: {{ include "common.names.fullname" . }}-secrets
               key: DATABASE_PASSWORD
```

Comment:
Let me know if I'm wrong, but I'm under the impression that the `if` statement on `postgresql.enabled` could be redundant. I thought the `airbyte.database.existingsecret.key` and `airbyte.database.existingsecret.key` are set to correct values thanks to definitions in `charts/airbyte/templates/_helpers.tpl` that already does the job of checking whether you enabled the PostgreSQL chart or not. 
I think it could ease readability if we could remove the `{{- if .Values.postgresql.enabled }}` from `deployment.yaml` file and leverage the helpers file to define the correct value according to deployment context.

Reply:
Yes, that's a fair point. I can move the logic to the helpers